Luke_Smith_Portfolio.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="Luke Smith Portfolio.css">
+
```

Comment:
This page does not have a title

Reply:
Also, the html validator's complaining that your filename has spaces in it. I don't think it has a functional impact on your website, but might be worth replacing them with underscores.